packages/contracts-bedrock/deploy-config/devnetL1-template.json

```diff
@@ -6,7 +6,7 @@
   "sequencerWindowSize": 200,
   "channelTimeout": 120,
   "p2pSequencerAddress": "0x9965507D1a55bcC2695C58ba16FB37d819B0A4dc",
-  "batchInboxAddress": "0xff00000000000000000000000000000000000901",
+  "batchInboxAddress": "0x00289C189bEE4E70334629f04Cd5eD602B6600eB",
```

Comment:
This is derived from the `l2ChainID` in this file (901) with the new standard approach used by the OPCM.

I elected to simply hardcode it rather than go through the exercise of deciding where to put the reusable logic, and whether to dedupe it by having the OPCM use a Library.